Fix duplicate link. Add note about inclusive language.

aphillips opened the pull request:
- Link checker flagged a duplicate link, fixed here.
- Pubrules warned about inclusive langauge use of the words "he", "his",
  and "her". I performed a complete search of the document. The terms
  "his" and "her" appear in the example of Hispanic names in section
  9.3.4. The term "he" also appears there. However, "he" appears
    throughout the document in the form of a language tag.

    To address, I added an HTML comment (not visible in the text) to
    remind ourselves in the future. We might consider providing an
    in-text note about inclusivity.


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/bp-i18n-specdev/pull/110.html" title="Last updated on Jul 25, 2023, 2:56 PM UTC (b26c991)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/110/33c87af...aphillips:b26c991.html" title="Last updated on Jul 25, 2023, 2:56 PM UTC (b26c991)">Diff</a>

aphillips committed:
Fix duplicate link. Add note about inclusive language.
- Link checker flagged a duplicate link, fixed here.
- Pubrules warned about inclusive langauge use of the words "he", "his",
  and "her". I performed a complete search of the document. The terms
  "his" and "her" appear in the example of Hispanic names in section
  9.3.4. The term "he" also appears there. However, "he" appears
    throughout the document in the form of a language tag.

    To address, I added an HTML comment (not visible in the text) to
    remind ourselves in the future. We might consider providing an
    in-text note about inclusivity.

aphillips committed:
Merge branch 'gh-pages' of https://github.com/aphillips/bp-i18n-specdev into gh-pages

r12a commented on index.html:
```diff
@@ -4,6 +4,13 @@
+      Per the W3C Manual of Style, this document avoids gender-specific pronouns.
+      In this document, the terms "he", "his", and "her" appear in an example of gender-specific
+      cultural norms (related to personal name handling in applications), which represents a
+      well-considered expection to this rule. The word-token "he"
```
expection -> exception

aphillips replied:
fixed

r12a commented on index.html:
```diff
@@ -36,7 +43,7 @@
     </script>
     <!-- local styles for this document -->
-	<link rel="stylesheet" href="local.css">
+	<link rel="stylesheet" data-import href="local.css">
```
For this to work, i think you need to include the following in respecConfig:

````
	postProcess: [
		async function importStyleSheet() { 
			const elems = document.querySelectorAll(`link[rel='stylesheet'][data-import]`)
			await Promise.all(
				[...elems].map(async link => {
					const text = await fetch(link.href).then(r => r.text())
					const style = document.createElement("style")
					style.textContent = text
					link.replaceWith(style)
					})
				)
			}
		],
````

aphillips replied:
fixed

aphillips committed:
Fix comments and errors
- Fix spelling error in the comment
- Add postProcess for local.css
- Fix broken link to css-counter-styles (it is required to be
  `css-counter-styles-3` by specref)

Note that the postProcess import of the stylesheet does not work on the
local copy, only from the repo, because of XSS prevention.

aphillips committed:
Fix character styling template and stylesheet reference

Having discussed it with Richard, updated the styles here along with all
of the text. This includes:

- Adding a link to respec_2022.css
- Removing some styles from local.css that are in that sheet
- Fix loading local.css
- Adding instructions to the local.css stylesheet

- Update the character naming styles
- Add a demo image (00A0.png) in an images directory
- Fix the echidna manifest for eventual publication
- Clean up the text per discussion with @r12a

aphillips commented:
@r12a Please have a look. I made the changes we just discussed here _first_, since this is supposed to be the source for the stuff in tr-design (and it impacts the linking stuff we also discussed).
